webservices/resources/download.py

```diff
@@ -12,16 +12,16 @@
 from webservices import exceptions
 from webservices.tasks import download
 from webservices.tasks import utils as task_utils
-from webservices.utils import use_kwargs
+from webservices.utils import use_kwargs_original
 
 client = boto3.client('s3')
 
-MAX_RECORDS = 100000
+MAX_RECORDS = 500000
```

Comment:
I should have called this out. My plan was to bump this up and test in `dev` first.